static/styles/zulip.css

```diff
@@ -449,56 +424,16 @@ a:hover code {
     top: -18px;
 }
 
-#streams_header a {
-    color: inherit;
-    text-decoration: none;
-}
-
-.tooltip {
-    max-width: 10em;
-}
-
-#stream_filters {
-    overflow: visible;
-    margin: 2px 0px 2px 0px;
-    padding: 0;
-    font-weight: normal;
-}
-
-#stream-filters-container {
-    overflow-y: hidden;
-    position: relative;
-}
-
-#stream-filters-container .ps-scrollbar-y-rail {
-    right: 0px !important;
-    width: 4px !important;
-}
-
-#stream-filters-container .ps-scrollbar-y {
-    width: 4px !important;
-}
-
-.stream-list-filter {
-    margin-left: 1ex;
-}
-
 .narrow-filter {
     display: block;
     position: relative;
 }
 
-#global_filters li:hover,
 #group-pms li:hover,
-#stream_filters li:hover,
 #user_presences li:hover {
     background-color: #e2e8dd;
 }
```

Comment:
A common file would be useful, especially since I'll be splitting the right sidebar next.